evm_loader/program/src/account_storage.rs

```diff
@@ -221,10 +221,10 @@ impl<'a> ProgramAccountStorage<'a> {
     }
 
     /// Get caller account info
-    pub fn get_caller_account_info(&self) -> Option<&AccountInfo<'a>> {
+    pub fn get_caller_account_info(&self) -> Option<(&AccountInfo<'a>, &AccountInfo<'a>)> {
         if let Some(account_index) = self.find_account(&self.origin()) {
-            let AccountMeta{ account, token: _, code: _ } = &self.account_metas[account_index];
-            return Some(account);
+            let AccountMeta{ account, token, code: _ } = &self.account_metas[account_index];
+            return Some((account, token));
```

Comment:
Do we really need token account to be returned?

Reply:
token was removed